src/inverseRobot.js

```diff
@@ -8,6 +8,20 @@
 
 function inverseRobot(robot) {
   // write code here
+  const fixRobot = {};
```

Comment:
Consider using a more meaningful name for the fixRobot variable, such as invertedRobot